submissions/ORybka/index.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://fonts.googleapis.com/css2?family=Patrick+Hand&display=swap" rel="stylesheet">
+  <link rel="shortcut icon" href="./assets/kitty.svg" type="image/svg">
+  <link rel="stylesheet" href="style.css">
+  <title>Hooli-popup</title>
+</head>
+<body>
+  <div class="container">
+    <header>
+      <nav class="navbar">
+        <ul class="menu">
+          <li class="menu-item">
+            <a href="#" >
+              Mail
+            </a>
+          </li>
+          <li class="menu-item">
+            <a href="#" >
+              Images
+            </a>
+          </li>
+          <li class="menu-item">
+            <input class="menu-label btn-open" type="checkbox" id="menu-btn">
```

Comment:
It's not a blocker, but maybe these class names aren't appropriate for the `input` tag.

Reply:
still have problems with names 😅